### ISSUE TITLE
typora is reformatting all Link Ref Defs even when they haven't been touched

### ISSUE THREAD

**Issue by vassudanagunta** [typora/typora-issues#5229]
Typora is reformatting all [Link Reference Definitions](https://spec.commonmark.org/0.30/#link-reference-definitions) even when they haven't been edited. It does even without editing the file. 

Specifically, it is replacing all `'` and `()` delimited [link titles](https://spec.commonmark.org/0.30/#link-title) with `"`delimited ones.

### steps to reproduce 
Start with the following source:
```markdown
CommonMark supports delimiting link titles in three ways.

Between straight double-quote characters:

[link title]: https://spec.commonmark.org/0.30/#link-title "link title spec"

Between straight single-quote characters:

[link title]: https://spec.commonmark.org/0.30/#link-title 'link title spec'

Beetween matching parentheses:

[link title]: https://spec.commonmark.org/0.30/#link-title (link title spec)
```
Open the file in Typora. View source.  Or press SAVE and then view the raw file. You will see that the last two link titles are reformatted to be wrapped by `"`.

## expectations

1. Typora should never modifying lines not edited by the user. 
2. Typora should never reformat elements unless requested by the user via command or settings.

These are important because:

- On of the most important reasons for using a plain text format is so content can be versioned and tracked just like code, often right alongside code. For example, using Git change tracking, diffs and merge. 
  - By making unnecessary changes, Typora is adding noise to version control history and version diffs. One small change on one line is resulting in many lines changed. 
  - Typora is forcing the user to manually and tediously revert dozens of changes as a workaround.
- intended (and valid) format of the source is violated. Sometimes compactness of source is important for a project.
- It breaks tools or processes that expect things in a particular format.
   - WebStorm uses a very specific format to insert TODOs in Markdown files, for TODO tracking and indexing:
      ```
      [//]: # (todo: rewrite this section)
      ``` 
  - Dev teams may have scripts that grep for patterns following code style conventions established by their team.

## related issues

- #1188
- #1189 
- #1316

**Comment by vassudanagunta** [comment]
This happens silently. I had no idea, until I noticed that my WebStorm TODO lists started getting shorter and shorter, even though I hadn't closed any. 

I had to go back and manually reformat over 60 TODOs spread across many files.

**Comment by abnerlee** [comment]
fixed in 1.3.6

**Comment by vassudanagunta** [comment]
thank you!
